terragrunt/aws/root/aws-organization/terragrunt.hcl

```diff
@@ -0,0 +1,8 @@
+terraform {
+  source = "../../../modules//aws-organization"
```

Comment:
```suggestion
  source = "../../../modules/aws-organization"
```

Reply:
The double slash is actually intentional. From the terragrunt docs:

> Terragrunt will download all the code in the repo (i.e. the part before the double-slash `//`) so that relative paths work correctly between modules in that repo.

Reply:
Oh. We should have a comment, then.

Reply:
This is actually a feature of Terraform: https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories

Reply:
@Mark-Simulacrum the `//` will be in every Terragrunt configuration file, would you expect a comment in each file?

Reply:
I guess I know about it now so maybe not so important. I think having a link in each file isn't terrible though.